File: Concurs Inot Java/IRepoParticipant.java
package me.repository;

import me.model.Participant;

import java.util.List;

public interface IRepoParticipant extends Repository<Integer, Participant> {
    List<Participant> findParticipantiDupaNume(String nume);
    List<Participant> findParticipantiDupaVarsta(int i);
}

File: Concurs Inot C#/IRepoParticipant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Problema_3_CSharp.me.model;

namespace Problema_3_CSharp.me.repository
{
    public interface IRepoParticipant : IRepository<int,Participant>
    {
        IEnumerable<Participant> FindParticipantiDupaNume(String nume);
        IEnumerable<Participant> FindParticipantiDupaVarsta(int i);
    }
}
